packages/nocodb/src/lib/meta/api/utilApis.ts

```diff
@@ -106,6 +109,16 @@ export async function axiosRequestMake(req: Request, res: Response) {
   return res.json(data?.data);
 }
 
+export async function axiosRequestMake(req: Request, res: Response) {
```

Comment:
The method is still vulnerable to SSRF since attackers can still call API via `curl` or other utility with an additional `type` value in the body with one of the values `csvUrl` or `excelUrl`